src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,71 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book() {
+
+    }
+
+    public Book(String title, BigDecimal price) {
+        this.title = title;
+        this.price = price;
+    }
```

Comment:
When you convert `ResultSet` to `Book`, it's better to create an object using setters or constructor but not both of them. Currently, you're using the constructor to set `title` and `price`, but you have a separate setter for `id`. It's not consistent to use both ways of initialization. Consider creating two constructors, one default and one with all the fields, or use setters for all fields.